true-index.html

```diff
@@ -7,7 +7,10 @@
 <div class="nav-hero-container">
     <div class="hero">
         <div class="container">
+            <h1 class="hero-title">Istio</h1>
+            <img class="hero-logo" src="{{home}}/img/istio-logo.svg" />
```

Comment:
does it have to have an alt attribute?
Also, can the html proofer run independent of the CLA check? I waited for it to run for half hour.. Nothing happened and I had to merge it out of expediency.